tests/test_host_check.py

```diff
@@ -486,14 +505,67 @@ def test_plat_specified_check_failing(self, capsys):
 ==================================================================
 !! Host NOT set up correctly for xrd-control-plane !!
 ==================================================================
+"""
+        assert output == cli_output
+        assert exit_code == 1
+
+    def test_plat_specified_check_erroring(self, capsys):
+        """Test a platform check erroring when host-check is run with arguments."""
+        exit_code, output = self.run_host_check(
+            capsys,
+            ["-p", "xrd-control-plane"],
+            erroring_checks=["Kernel version"],
+            failing_checks=["Kernel version"],
```

Comment:
This makes less sense than the core pattern check failing - a single check can't both fail and error.